### DISCUSSION TITLE
Question about the results

### DISCUSSION THREAD

**Discussion by AndyLy2Zy** [ablab/spades#1114] [General]
I used the spades to de novo a mutant yeast genome. After that, I blast the scaffolds against the reference genome from NCBI. I found that the scaffolds are same with the reference completely.  did there have any suggestion?

<img width="461" alt="69c26dcacae2421dcdc9c0ec9f79d69" src="https://user-images.githubusercontent.com/115370378/227745220-be749e53-159d-49aa-876f-bf8140bc3dd8.png">

**Comment by asl** [comment]
Well, likely your species in question is known and not a mutant at all.
